tests/dummy/config/ember-try.js

```diff
@@ -40,6 +40,18 @@ module.exports = async function () {
       },
       embroiderSafe(),
       embroiderOptimized(),
+      {
+        name: 'ember-concurrency-v3',
+        npm: {
+          dependencies: {
+            'ember-concurrency': '^3.0.0',
+          },
+          devDependencies: {
+            'ember-basic-dropdown': null,
```

Comment:
We only installed this because ember-power-select v8 has it listed as a peerDependency. In v7 it's bundled as a dependency.